app/presenters/track.rb

```diff
@@ -47,17 +47,17 @@ def topic_content(topic_key)
       end
 
       def contributing_instructions(topic_key)
-        content_with_trackler_data("./x/docs/md/track/BETTER.md").gsub('TOPIC', topic_key.to_s.upcase)
+        populate_template_with_trackler_data(File.read("./x/docs/md/track/BETTER.md")).gsub('TOPIC', topic_key.to_s.upcase)
```

Comment:
Thanks, I see.

I suspect this distinction is important. I wonder if there is a way to make it more clear in the code. The last few refactorings have been pushing the implementations closer together, but that may have been the wrong thing to be doing.

Do you have any suggestions?
("Don't worry so much about it" is valid :wink: )